hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/rollback/SparkMarkerBasedRollbackStrategy.java

```diff
@@ -52,6 +52,7 @@ public SparkMarkerBasedRollbackStrategy(HoodieTable<T, JavaRDD<HoodieRecord<T>>,
       MarkerFiles markerFiles = new MarkerFiles(table, instantToRollback.getTimestamp());
       List<String> markerFilePaths = markerFiles.allMarkerFilePaths();
       int parallelism = Math.max(Math.min(markerFilePaths.size(), config.getRollbackParallelism()), 1);
+      jsc.setJobGroup(this.getClass().getSimpleName(), "Marker files rollback");
```

Comment:
Should we clear the jobGroup after the job finish?If not, the next job will reuse the group name.

Reply:
@pengzhiwei2018 Since each job should have been set their own job group name, so it should be ok?

Reply:
> Should we clear the jobGroup after the job finish?If not, the next job will reuse the group name.

@pengzhiwei2018 Some method to do one specific operation of hudi, may be contain not only one spark action. it can reuse the jobGroup. In this scenario , We don’t have to set it so detailed. Also it will be more suitable just like @leesf mention,  follow the original mechanism of Hudi.

Reply:
@leesf @lw309637554 Maybe we can make all of the spark job in hoodie have a job group name. But if user does some spark action after the hoodie job.The user's  job may reuse the hoodie's job group name.It is hard to control the user's behavior.

Reply:
yeah, as your case  you can set job group in you spark job. just like hudi do .

Reply:
may be we can set the jobGroup to `""` upon writeClient.close() or something?

Reply:
this option make sense